user-guide/requests-list.md

```diff
@@ -9,51 +9,54 @@ position: 60
 
 # Requests List
 
-The **Requests** list in Fiddler Everywhere provides the option to save, modify, execute, and share previously composed requests. The [Composer](https://docs.telerik.com/fiddler-everywhere/user-guide/composer) tab enables you to build and send HTTP and HTTPS request manually. When you **save** a request using the Composer, the request is added to a **collection** in the **Requests** list. 
+The **Requests** list in Fiddler Everywhere provides the option to save, modify, execute, and share previously composed requests. When you **save** a request using the [Composer](https://docs.telerik.com/fiddler-everywhere/user-guide/composer), the request is added to a **collection** in the **Requests** list.
+
+This article describes how to organize and modify existing requests. To learn more about building and sending HTTP and HTTPS requests, see the [Composer](https://docs.telerik.com/fiddler-everywhere/user-guide/composer) article.
```

Comment:
There are several topics that we mention in the introductory paragraph. That's why it's important to note that we don't cover them in this article.